{"_id":"70a07b2c-89e4-3759-832e-1de84d202b3f","authorName":"Anonymous","authorEmailEncrypted":"SFwUGMhlmW8ksSrY4Yy0JbOWdSXw+jD7qNHaacQpGGfBB2PMlvcU6nIY7e1l3gG6QnYWTRH/u0ydagfCJNWFhlGa/FlcIGq8018bRJWq0KVW50gIBpk5ncJ3mLl61nrsvU8qIudd3h5ZZgadyfTMHTU3+Z8o2ZFhEg8s92/TL5TPLU8dKSP1wxx6Pojccgi4hPHQNK31mGkCJetg4n+qgRzZP1bDmXDXJcfY35c3VUDwJKElUihXJXHnohyddumv6YiZ7Un81TCbi8Xf6xbg3a/zXkdtpOcyTJDvYPLFMbkoML/u24cOsONUIys8FoSYbGeGTxMFBKnzb+PbaM7aaHNeqZy3DMJ0eSM+l3Ur5lAkiMaBJ9Zmm5Ny6HbHdE6kWAW7fkkwI6wf1D6UYjooluWxE7wTV7kiW6FvkMCBLYpnEmTeNlsD0mUGWFjo4v4aHMo5pmaiMrxUlQKc8kfIHujlLSTZKIcHTDVsW3/8y4FBUQslQZpz1DeWFBYcyleTQk1YbWahp4sJVgbS1qmdwoVXW8a9ZWLlZamQTckQQ1XicB3aVc4dDvmVkmTZHDKVmqIEIlDusjQoZqXmfeB1K9aq5EkWKmsAp7W2aAgfntXPNsZouprVwOc+Qo1doN0NpTJSITbatlfBqhislJUJQa2OEJPWjAf6BfmQ+VY1hLA=","authorEmailMD5":"3cb42db01b5edab948a576e1bf94a281","message":"I'm not sure cancellation as an exception is a good thing. There are valid reasons why you may want to cancel a task, and therefore wouldn't want an exception thrown. In fact the more I think about it the worse that sounds.","postId":"2013_09_taskrun-vs-backgroundworker-round-4-e33eb673-a094-3715-9282-0e28e7b5e733","replyTo":"","date":"2013-09-13T12:57:03Z","timestamp":1379077023,"authorUserId":""}